### PR TITLE
Create Jolly Dark.theme.json

### PR DESCRIPTION
**Pull request by nicolasmtjt**
A simple theme, with few colours, highlighting mostly links and footnotes. Text, titles, and blockquotes are different shades of light gray.

### CODE REVIEW

**Comment by DivineDominion** [comment]
@nicolasmtjt Thanks for this theme and #16!

Could you attach a screenshot for each real quick to the PRs for comparison and future reference?

(In my book, a dark mode screenshot with the dark theme, and a light mode screenshot with the light theme would be amazing. No need for all 4 permutations)

File: Themes/Jolly Dark.theme.json
Comment (nicolasmtjt) [inline Themes/Jolly Dark.theme.json:1]: Screenshot attached
Jolly-dark-screenshot

**Review by nicolasmtjt — COMMENTED** [review]
Added screenshot

**Comment by nicolasmtjt** [comment]
Thank you!

**Pull request merged**